src/hotspot/share/c1/c1_LIR.cpp

```diff
@@ -510,9 +510,9 @@ void LIR_OpVisitState::visit(LIR_Op* op) {
       if (opBranch->_opr1->is_valid()) do_input(opBranch->_opr1);
       if (opBranch->_opr2->is_valid()) do_input(opBranch->_opr2);
 
-      if (opBranch->_info != NULL)     do_info(opBranch->_info);
+      if (opBranch->_info != nullptr)     do_info(opBranch->_info);
```

Comment:
Maybe keep previous alignment:
```suggestion
      if (opBranch->_info != nullptr)  do_info(opBranch->_info);
```